tests/cmd/list-targets/env.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
i don't know if this is the env file we want in our test suite, but it's the one @rmustacc gave me, and it works, so...